SmartDeviceLink/SDLTouchManager.h

```diff
@@ -0,0 +1,33 @@
+//
+//  SDLTouchManager.h
+//  SmartDeviceLink-iOS
+//
+//  Created by Muller, Alexander (A.) on 6/14/16.
+//  Copyright © 2016 smartdevicelink. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "SDLTouchManagerListener.h"
+
+@interface SDLTouchManager : NSObject
+
+@property (nonatomic, weak) id<SDLTouchManagerListener> touchEventListener;
+
+/*
+ *  Distance between taps.
+ */
+@property (nonatomic) CGFloat tapDistanceThreshold;
```

Comment:
Explicitly specify as assign(?), even though it's obvious, just for consistency. Atomicity and strong/weak/assign/copy should be on every property.